src/apps/statements/repo/modelsRepo/getStatements/memory/matchesCursorOption.test.ts

```diff
@@ -0,0 +1,31 @@
+import * as assert from 'assert';
```

Comment:
Thanks for testing this, I think we should probably remove all of the memory repo logic at some point to simplify this repo. Whilst the memory repo is quite helpful for running the tests on the business logic quicker, it's never actually used in reality.